mesonbuild/backend/vs2010backend.py

```diff
@@ -691,7 +699,8 @@ def create_basic_project(self, target_name, *,
             if target_ext:
                 ET.SubElement(direlem, 'TargetExt').text = target_ext
 
-            ET.SubElement(direlem, 'EmbedManifest').text = 'false'
+            # Fix weird mt.exe error
```

Comment:
What weird error? I think it would be better to remove this comment, and to fix the next line in a separate commit explaining what it is fixing.

Reply:
mt.exe is trying to compile a non-existent `.generated.manifest` file and link it with the target if this element exists. This does not happen without masm props.